<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="INPOD" name="Create QB" tests="1" failures="1" timestamp="12 Aug 2016 12:12:06 GMT" time="1.342" errors="0">
  <testcase name="UploadQb" time="1.329" classname="Teacher.UploadQB">
    <failure type="java.lang.AssertionError" message="Unable TO create QB">
      <![CDATA[java.lang.AssertionError: Unable TO create QB
	at Teacher.UploadQB.UploadQb(UploadQB.java:62)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- UploadQb -->
</testsuite> <!-- Create QB -->
